### ISSUE TITLE
ASC log header has duplicate lines

### ISSUE THREAD

**Issue by bauer-san** [hardbyte/python-can#379]
The `base hex  timestamps absolute` line is written twice by the ASC log writer.  Line 136 in __init__ and 166 are each writing these lines.  The resultant .ASC file cannot be read by Vector tools.

```
date Tue Jul 07 09:49:50 AM 2018
base hex  timestamps absolute
internal events logged
base hex  timestamps absolute
Begin Triggerblock Tue Jul 07 09:49:50 AM 2018
```

**Comment by felixdivo** [comment]
Yes, I see the problem in the code. I could if fix it as part of the huge PR #348, but I need to know whether the first or the last line with `base hex ...` is wrong.

**Comment by felixdivo** [comment]
In the example in `python-can/test/data/logfile.asc`, the second line is missing.

**Comment by bauer-san** [comment]
Exactly right.  
I should have stated the expected behavior in the original issue.

**Comment by felixdivo** [comment]
No problem. I'll fix it right now, and the patch will be available on the next release after #348 got merged.
